R/calc_aggregate_counts.R

```diff
@@ -80,17 +136,23 @@ calc_aggregate_counts <- function(
         ))
 
     if(state){
-        return(state_df)
+        return(harm_df)
     }
 
-    agg_df <- state_df %>%
+    agg_df <- harm_df %>%
         filter(!is.na(Val)) %>%
-        group_by(Measure) %>%
+        group_by(Measure)
+
+    if(all_dates){
+        agg_df <- group_by(agg_df, Date, Measure)
+    }
+
+    out_agg_df <- agg_df %>%
         summarize(
             Count = sum_na_rm(Val), Reporting = sum(!is.na(Val)),
```

Comment:
When do you expect to see a difference between sum_na_rm(Val) and sum(!is.na(Val)) ?

Reply:
Yeah `sum_na_rm(Val)` gives you the sum of all the non NA counts for a metric where as `sum(!is.na(Val))` tells you how many agencies report any value for a given metric, it can only be at most 52.

Reply:
That makes total sense, thank you for breaking it down for me :)